templates/vimeoTranscript.hbs

```diff
@@ -0,0 +1,32 @@
+<div class="media-transcript-button-container">
+    {{#if _inlineTranscript}}
+        <button class="media-inline-transcript-button button" aria-expanded="false">
+            <div class="transcript-text-container">
+                {{#if inlineTranscriptButton}}
+                    {{inlineTranscriptButton}}
+                {{else}}
+                    {{transcriptLink}}
+                {{/if}}
+            </div>
+        </button>
+    {{/if}}
+    {{#if _externalTranscript}}
+        <button onclick="window.open('{{transcriptLink}}')" class="media-external-transcript-button button">
+            <div class="transcript-text-container">
+                {{#if transcriptLinkButton}}
+                    {{transcriptLinkButton}}
+                {{else}}
+                    {{transcriptLink}}
+                {{/if}}
+            </div>
+        </button>
+    {{/if}}
+</div>
+
+{{#if _inlineTranscript}}
+    <div class="media-inline-transcript-body-container" role="region" aria-label="{{#if inlineTranscriptButton}}{{inlineTranscriptButton}}{{else}}{{transcriptLink}}{{/if}}">
```

Comment:
the `else` here needs removing as the logic is currently that if there's no inlineTranscriptButton defined use the link to the external transcript as the ARIA label for the inline transcript div... can't see how that would ever something you'd actually want!

 (this is also an issue in the adapt media component which will be sorted as part of v5)